tests/ci/run_benchmark_build_tests.sh

```diff
@@ -23,25 +23,27 @@ function build_aws_lc_fips {
       -DFIPS=1 \
       -DENABLE_DILITHIUM=ON \
       -DCMAKE_BUILD_TYPE=RelWithDebInfo \
+      -DBUILD_SHARED_LIBS=1 \
       -DBUILD_TESTING=OFF
   pushd "$BUILD_ROOT"
   ninja install
   popd
 }
 
-function build_aws_lc_fips_2022 {
-    echo "building the fips-2022-11-02 branch of aws-lc in FIPS mode"
-    git clone --depth 1 --branch fips-2022-11-02 https://github.com/aws/aws-lc.git "${scratch_folder}/aws-lc-fips-2022-11-02"
-    pushd "${scratch_folder}/aws-lc-fips-2022-11-02"
+function build_aws_lc_branch {
+    branch="$1"
+    echo "building the ${branch} branch of aws-lc in FIPS mode"
+    git clone --depth 1 --branch $branch https://github.com/aws/aws-lc.git "${scratch_folder}/aws-lc-${branch}"
+    pushd "${scratch_folder}/aws-lc-${branch}"
     cmake -GNinja \
-        -DCMAKE_INSTALL_PREFIX="${install_dir}/aws-lc-fips-2022-11-02" \
+        -DCMAKE_INSTALL_PREFIX="${install_dir}/aws-lc-${branch}" \
         -DFIPS=1 \
         -DENABLE_DILITHIUM=ON \
         -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-        -DBUILD_TESTING=OFF
```

Comment:
we still don't need testing, right?

Reply:
I forgot why I turned it back on, turns out the [old FIPS branch can't handle turning off the tests and turn FIPS on](https://us-west-2.codebuild.aws.amazon.com/project/eyJlbmNyeXB0ZWREYXRhIjoiWk5IUWJGRGxBcTJ2Mkp4WGF3dnBwYjc5V0ZZYSt5SVVGbkwvODkydTNTaVQ2V2FMN3hwa0tjSWNFemw2QWtCWW5welFWV3lpRFpKVitwejgvelhpRWh3NDNqcWhKalpPYW9hL2tLMDlJSDFPT1NkNyIsIml2UGFyYW1ldGVyU3BlYyI6Ilp4VXRNQXFGM1BZYVlaRkIiLCJtYXRlcmlhbFNldFNlcmlhbCI6MX0%3D/build/70dc6688-29ad-489e-8d17-0b6223c4c0a4). It's just a cmake issue that we didn't test/fix on the old branch.